src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -63,13 +64,15 @@ internal sealed class JavaScriptAnalyzer : IAnalyzer, IDisposable
             ITelemetryManager telemetryManager,
             IAnalysisStatusNotifier analysisStatusNotifier,
             IActiveSolutionTracker activeSolutionTracker,
+            IAnalysisConfigMonitor analysisConfigMonitor,
```

Comment:
The CTOR of this class is getting ginormous but I think there's a way to do some tidying up: a lot of the dependencies are part of the EslintBridgeProcess's management and initialization of the EslintBridgeClient; I think we can extract `EnsureEslintBridgeClientIsInitialized` to another class that returns an initialized instance of the client + maintains the lifetime of the process. We could make it generic and re-use in TypescriptAnalyzer.
I suggest we take a look at this when we write the TS stuff.